src/main/java/de/oliver/fancyholograms/HologramManagerImpl.java

```diff
@@ -58,10 +58,21 @@ public final class HologramManagerImpl implements HologramManager {
      */
     @Override
     public @NotNull
-    @UnmodifiableView Collection<Hologram> getHolograms() {
+    @UnmodifiableView Collection<Hologram> getAllHolograms() {
         return Collections.unmodifiableCollection(this.holograms.values());
     }
 
+    /**
+     * Returns a read-only view of the currently loaded persistent holograms.
+     *
+     * @return A read-only collection of holograms.
+     */
+    @Override
+    public @NotNull
+    @UnmodifiableView Collection<Hologram> getHolograms() {
+        return this.holograms.values().stream().filter(hologram -> hologram.getData().isPersistent()).toList();
+    }
+
```

Comment:
I think I would personally name this `getPersistentHolograms` and then revert `getAllHolograms` to `getHolograms`.

Reply:
Awesome, I think the only thing left would be to look at those methods names mentioned above